src/components/Editor/Editor.js

```diff
@@ -584,6 +594,11 @@ class Editor extends React.Component {
                 />
               )}
             </label>
+            {this.props.wordCount === 'on' &&
+              <label htmlFor="reading_time" className="Editor__readingTime">
+                <FormattedMessage id="reading_time" defaultMessage={`${readingTime(this.state.contentHtml).words} Words / ${readingTime(this.state.contentHtml).text}`} />
```

Comment:
`reading-time(...).text` is not translatable. We should make it translatable some way.